src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/repository/TaskResourceCustomRepositoryImpl.java

```diff
@@ -103,7 +105,7 @@ private String extraConstraints(List<String> excludeCaseIds, SearchRequest searc
         if (searchRequest.isAvailableTasksOnly()) {
             extraConstraints.append("AND assignee IS NULL ");
         } else {
-            extraConstraints.append(buildListConstraint(searchRequest.getUsers(), "assignee", "assignee", true));
+            extraConstraints.append(buildListConstraint(searchRequest.getUsers(), ASSIGNEE, ASSIGNEE, true));
```

Comment:
Can I suggest a convention for naming some of these?  
Maybe if it's a DB column name we can call it something like `COL_ASSIGNEE` or `DB_COL_ASSIGNEE`?